src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractTypeImportCompletionService.cs

```diff
@@ -33,13 +34,24 @@ internal abstract partial class AbstractTypeImportCompletionService : ITypeImpor
         protected abstract string Language { get; }
 
         internal AbstractTypeImportCompletionService(Workspace workspace)
-            => CacheService = workspace.Services.GetRequiredService<IImportCompletionCacheService<CacheEntry, CacheEntry>>();
+        {
+            _workspace = workspace;
```

Comment:
do you actually need to store the workspace around anymore?

Reply:
remoevd